Committees/National_Political_Education_Committee.adoc

```diff
@@ -0,0 +1,29 @@
+= National Political Education Committee
+
+== Purpose
+
+Political education of PLTCom members is the sole mission of this committee. It will provide all members a course in capitalism first and scientific socialism second followed by a course on party work. It will also maintain a comprehensive digital library of marxist political material as well as a physical library as possible. 
+
+== Structure
+
+Only _full_ members can hold these positions:
+
+- Committee Chair
+- Librarian
+- Teacher(s)
+
+== Meetings
+
+This committee will meet at a minimum of 4 times per month to discuss and plan educational curriculum. 

```

Comment:
Firearms committee meets monthly, should they be the same for consistency's sake?